### PR TITLE
bin: add flag to print version information

### PR DESCRIPTION
**Pull request by Librazy**
for #1239

### CODE REVIEW

**Comment by siddontang** [comment]
Thanks @Librazy 

Please sign the CLA first.

File: src/util/mod.rs [hunk 1]
```diff
@@ -387,7 +387,7 @@ impl<L, R> Either<L, R> {
 }
 
 /// `build_info` returns a tuple of Strings that contains build utc time and commit hash.
-fn build_info() -> (String, String) {
+pub fn build_info() -> (String, String) {
```
Comment (BusyJay) [inline src/util/mod.rs]: Why not just call `print_tikv_info` instead?

Comment (siddontang) [inline src/util/mod.rs]: `print_tikv_info` may take some `log` information like file, number, log level.

Comment (Librazy) [inline src/util/mod.rs]: print_tikv_info use info! marco to print the version info, which would introduce some noise like timestamps and line numbers. And we had not initialize the log system yet (at tikv-server.rs:893 but initial_log at tikv-server.rs:910)

Comment (Librazy) [inline src/util/mod.rs]: the -v/--version flag is just for the convenience of cli or bash script. And we expect the -v flag be used alone (like the -h flag). So we don't need to bother the log system but do some println!.

**Comment by siddontang** [comment]
LGTM

**Comment by siddontang** [comment]
Thanks @Librazy 

Could you help us add this for https://github.com/pingcap/pd :-)

**Comment by Librazy** [comment]
My pleasure.

**Pull request merged**